package.json

```diff
@@ -30,14 +30,14 @@
   "dependencies": {},
   "peerDependencies": {
     "axe-core": "^3 || ^4",
-    "cypress": "^3 || ^4 || ^5 || ^6"
+    "cypress": "^3 || ^4 || ^5 || ^6 || ^7 || ^8"
```

Comment:
Alternatively, this could be `>= 3` but I didn't want to jump to this without contributor feedback